userguide/install_kernel_modules.rst

```diff
@@ -0,0 +1,71 @@
+Install Kernel Modules
+======================
+
+Install with DKMS
+^^^^^^^^^^^^^^^^^
+
+In order to support the mandatory kernel subsystems ashmem and binder for the
+Android container you have to install two
+`DKMS <https://en.wikipedia.org/wiki/Dynamic_Kernel_Module_Support>`_
+based kernel modules. The source for the kernel modules is maintained by the
+Anbox project `here <https://github.com/anbox/anbox-modules>`_.
+
+At the moment we only have packages prepared for Ubuntu in a PPA on Launchpad.
+If you want to help to get the packages in your favorite distribution please
+come and talk to us or submit a PR with the distribution specific packaging.
+
+Install DKMS package from PPA
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In order to add the PPA to your Ubuntu system please run the following commands:
+
+.. code-block:: text
+
+    $ sudo add-apt-repository ppa:morphis/anbox-support
+    $ sudo apt update
+    $ sudo apt install linux-headers-generic anbox-modules-dkms
```

Comment:
While we're on it, installing `linux-headers-generic` doesn't seem to match the case where people have installed the HWE kernel or any other. I am not sure what the best way here is to get automatically installed the right kernel headers for the current kernel version all the time so that even this still works on update. Any idea?

Reply:
There're many kernel versions, people should install the generic/meta one. Otherwise after upgrading, they may miss the headers.